src/app/main/component/places/places.component.ts

```diff
@@ -68,10 +69,12 @@ export class PlacesComponent implements OnInit, OnDestroy {
     private favoritePlaceService: FavoritePlaceService,
     private googleScript: GoogleScript,
     private dialog: MatDialog,
-    private userOwnAuthService: UserOwnAuthService
+    private userOwnAuthService: UserOwnAuthService,
```

Comment:
[Member 'userOwnAuthService: UserOwnAuthService' is never reassigned; mark it as `readonly`.](https://sonarcloud.io/project/issues?pullRequest=3407&open=AZKVdKJqhY9BiVF5AhU5&id=ita-social-projects-green-city-client)